Buildings/Resources/Scripts/Dymola/Experimental/DHC/Examples/Cooling/CoolingSystem.mos

```diff
@@ -0,0 +1,35 @@
+﻿// Simulate
+simulateModel("Buildings.Experimental.DHC.Examples.Cooling.CoolingSystem", stopTime=86400, method="Cvode", tolerance=1e-06, resultFile="CoolingSystem");
+// Plot commands
+removePlots(false);
+createPlot(
+    id=1, 
+    position={20, 4, 591, 536}, 
+    y={"pla.port_a.p", "pla.port_b.p", "bld[1].p_rel", "bld[2].p_rel", "bld[3].p_rel"}, 
```

Comment:
I expect the number of buildings here should also match `nBui` in `CoolingSystem.mos`. Please adjust accordingly.

Reply:
Matched the plot settings with the value of `nBui`.